linkis-dist/pom.xml

```diff
@@ -218,4 +218,87 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>docker</id>
+            <properties>
+                <image.build.type>dev</image.build.type>
+                <image.base>centos:7</image.base>
+                <image.base.web>nginx:1.19.6</image.base.web>
+                <jdk.version>1.8.0-openjdk</jdk.version>
+                <jdk.build.revision>1.8.0.332.b09-1.el7_9</jdk.build.revision>
+                <linkis.system.user>hadoop</linkis.system.user>
+                <linkis.system.uid>9001</linkis.system.uid>
+                <linkis.home>/opt/linkis</linkis.home>
+                <linkis.conf.dir>/etc/linkis-conf</linkis.conf.dir>
+                <linkis.log.dir>/var/logs/linkis</linkis.log.dir>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>build-linkis-image</id>
+                                <phase>install</phase>
```

Comment:
I think the image build should not be include in the 'install' phase automatically, we can add some doc in project `https://github.com/apache/incubator-linkis-website`

Reply:
has add some doc in the README.md, use the -Pdocker to enable the process. I will add a detail doc in incubator-linkis-website in later PR.